### ISSUE TITLE
Further accessbility improvements to search result titles

### ISSUE THREAD

**Issue by zackkrida** [WordPress/openverse#2150]
## Problem

Currently search result titles announce the type of media (currently image or audio) and the title of a work. Visually, however, users are privy to much more information, including the CC licenses.

## Description

We may want to announce what amounts to the full CC attribution for these works, including the license info and author name.

## Alternatives

Conversely, this may be too much information for users and detract from the experience of moving through the results quickly to find a suitable image.

**Comment by dhruvkb** [comment]
I am _not an accessibility expert_ but imo, the exact same amount of information as accessible to screen readers as to a person reading the screen directly. So while I'm pro-"adding more info to the SR text", I'm anti-"full attribution in the SR text".

**Comment by dhruvkb** [comment]
Added the "discussion" and "ticket work required" labels till we finalise what the exact text should be.

**Comment by sarayourfriend** [comment]
I agree with @dhruvkb. SR accessibility doesn't necessarily mean providing the exact same full context so long as there are other ways of obtaining the same information. @zackkrida Did you get this feedback from someone based on experience using the site from a screen reader? As Dhruv and you've both pointed out, we don't want to make the site _harder_ to navigate for someone using a screen reader. Perhaps making license _filters_ more readily accessible is the best way to make sure screen reader users know the context of specific results rather than fully describing the licenses on each result.

**Comment by zackkrida** [comment]
I think we should add "{media} on Openverse" to the titles, we don't need to include any other information. Just the most basic amount of content to let you know you're visiting the page of a single, named work.

**Comment by zackkrida** [comment]
I'm actually going to close this and create a separate issue for the problem described in [this comment](https://github.com/WordPress/openverse/issues/2150#issuecomment-2243264930), which refers to single result page SEO titles. The original issue, I believe, was referring to the individual result titles on a search result page, and how they should be presented. In that case I think sharing the title is sufficient, and delving into the single result can be used to determine the specific license and suitability of the result for use.
